style_guide.md

```diff
@@ -0,0 +1,118 @@
+
+In order to keep things beginner friendly... 
+* All classes, methods, and functions should include a docstring. (Sec. 3)
+* Loop and control structures should be documented at each line. (Sec. 4)
+
+## 2. Names
+
+### Best Naming Practices
+Names should be **descriptive**, and **concise**.
+*  **Descriptive**: Names should accurately describe the entity they represent. 
+*  **Concise**: Names should be as short as possible without extraneous abbreviation, or loss of meaning. 
+
+"Function" in this section refers to a functions, methods, and generators as group.
+
+### Names of Structures and Data
+Names should follow [Google Python](http://google.github.io/styleguide/pyguide.html?showone=Naming#Naming) convention. Simply put...
+* **Classes** should be upper camel case, e.x. ```class MySuperCoolClass():```
+* **Functions**: should be lowercase underscored, e.x. ```def my_function():``` 
+* **Variables**: should be lowercase underscored, e.x. ```my_integer```
+
+## 3. Comments
+Some of our guidelines are contradictory to [PEP 8](https://www.python.org/dev/peps/pep-0008/#introduction) and [Google Python](http://google.github.io/styleguide/pyguide.html?showone=Comments#Comments) standards.
+We deviate from these formats in an effort to remain **beginner friendly**.
+
+### Inline Comments
+You should document all loop and control structures with inline comments. 
+These comments should be indented at least 8 spaces after the end of the line. 
+Indentation of the comment should match indentation of the line it describes. 
+There should be a space after each '#', and the first letter of the comment should be captialized. 
+Ex. 
+```python
+while self.isAlive is True:                # While person is alive
+    if self.age < 6 or self.isRetired:         # If too old or young
+        stay_home()                                # Stay home
+    elif self.age < 18:                        # Else if school age
+        goto_school()                              # Goto school
+    else:                                      # Otherwise
+        goto_work()                                # Goto work
+```
+
+### Block Comments 
+Block comments should be used to describe logical groupings of statements, if appropriate, but not loop or control structures. 
+They should be placed above the code block they describe, with a new line above itself and below the code block it describes. 
+Ex. 
+```python
+# Do the work
+work_it_harder()
+make_it_better()
+do_it_faster()
+makes_us_stronger()
+
+# Ramble on afterwards
+more_than_ever()
+hour_after()
+our_work_is()
+never_over()
+
+end_song()
+```
+## 4. Docstrings
+Docstrings should follow the [Google Python](http://google.github.io/styleguide/pyguide.html?showone=Comments#Comments) format.
+
+### Single Line Summaries
+If you are having trouble writing a single line summary in a single line; the reason usually boils down to **"This X is too complicated."** 
```

Comment:
Before your text, maybe we can add the sentence:
Docstrings always begin with a Singe Line Summary. If your are having...